config_spec.toml

```diff
@@ -93,11 +93,17 @@ type = "f32"
 doc = "Total size of block transactions IDs to cache (in MB)"
 default = "10.0"
 
+[[param]]
+name = "txid_warning_limit"
+type = "usize"
+doc = "Number of transactions to lookup before returning a warning that the amount is getting close to the true limit; See txid_limit below"
+default = "250"
+
```

Comment:
What is the use of this warning? If an address has too many transactions, the query will fail, if it hasn't it will succeed.